src/i18n/locales/en-US.json

```diff
@@ -83,6 +83,26 @@
   "components.login.appstartscreen.loginButton": "Login",
   "components.login.custompinlogin.title": "Enter PIN",
   "components.ma.assetSelector.placeHolder": "Select an asset",
+  "components.nftDetails.title": "NFT Details",
+  "components.nftDetails.overview": "Overview",
+  "components.nftDetails.metadata": "Metadata",
+  "components.nftDetails.nftName": "NFT Name",
+  "components.nftDetails.createdAt": "Created",
+  "components.nftDetails.description": "Description",
+  "components.nftDetails.author": "Author",
+  "components.nftDetails.fingerprint": "Fingerprint",
+  "components.nftDetails.policyId": "Policy id",
+  "components.nftDetails.detailsLinks": "Details on",
+  "components.nftDetails.copyMetadata": "Copy metadata",
+  "components.nftDetailsNavigator.title": "NFT Details",
+  "components.nfts.noNFTs": "No NFTs found",
+  "components.nfts.nftCount": "NFT count",
+  "components.nfts.errorTitle": "Oops!",
+  "components.nfts.errorDescription": "Something went wrong.",
+  "components.nfts.reloadApp": "Try to reload this page or restart the app.",
+  "components.nftsNavigation.title": "NFT Gallery",
+  "components.nftsNavigation.search": "Search NFT",
+  "components.common.navigation.nftGallery": "NFT Gallery",
```

Comment:
move to `nft.*` this component.component.component for translations is a bad practice